[2024-11-12T22:49:51.545+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-12T22:49:51.582+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extractAPItoBronze.get_api manual__2024-11-12T21:23:43.667318+00:00 [queued]>
[2024-11-12T22:49:51.606+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extractAPItoBronze.get_api manual__2024-11-12T21:23:43.667318+00:00 [queued]>
[2024-11-12T22:49:51.608+0000] {taskinstance.py:2303} INFO - Starting attempt 23 of 24
[2024-11-12T22:49:51.647+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): get_api> on 2024-11-12 21:23:43.667318+00:00
[2024-11-12T22:49:51.656+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=888) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-12T22:49:51.658+0000] {standard_task_runner.py:63} INFO - Started process 890 to run task
[2024-11-12T22:49:51.659+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'extractAPItoBronze', 'get_api', 'manual__2024-11-12T21:23:43.667318+00:00', '--job-id', '149', '--raw', '--subdir', 'DAGS_FOLDER/ExtracttoBronze.py', '--cfg-path', '/tmp/tmplj_dcnsn']
[2024-11-12T22:49:51.664+0000] {standard_task_runner.py:91} INFO - Job 149: Subtask get_api
[2024-11-12T22:49:51.732+0000] {task_command.py:426} INFO - Running <TaskInstance: extractAPItoBronze.get_api manual__2024-11-12T21:23:43.667318+00:00 [running]> on host bbd2fa8da241
[2024-11-12T22:49:51.864+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extractAPItoBronze' AIRFLOW_CTX_TASK_ID='get_api' AIRFLOW_CTX_EXECUTION_DATE='2024-11-12T21:23:43.667318+00:00' AIRFLOW_CTX_TRY_NUMBER='23' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-12T21:23:43.667318+00:00'
[2024-11-12T22:49:51.866+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-12T22:49:51.886+0000] {logging_mixin.py:188} INFO - https://api.openbrewerydb.org/breweries
[2024-11-12T22:50:13.866+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-12T22:50:13.870+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/ExtracttoBronze.py", line 40, in processing_api_json
    if df.empty:
       ^^
NameError: name 'df' is not defined
[2024-11-12T22:50:13.915+0000] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=extractAPItoBronze, task_id=get_api, execution_date=20241112T212343, start_date=20241112T224951, end_date=20241112T225013
[2024-11-12T22:50:13.921+0000] {logging_mixin.py:188} INFO - b'\x9bJ\xb79\x1a \xfd\xf6\x066Xz\x8aW2\xfe\xc0\x8ds(\xd9%8\xc3\xe2\xa0\x83\xa7\x8e\xf9\xa6\x02'
[2024-11-12T22:50:13.928+0000] {logging_mixin.py:188} INFO - Senha final (sem padding): b'epcrcnckxddwcogu'
[2024-11-12T22:50:13.929+0000] {logging_mixin.py:188} INFO - epcrcnckxddwcogu
[2024-11-12T22:50:21.221+0000] {logging_mixin.py:188} INFO - Failed to send email: (535, b'5.7.139 Authentication unsuccessful, basic authentication is disabled. [CP2PR80CA0249.lamprd80.prod.outlook.com 2024-11-12T22:50:19.680Z 08DD02E90BFF2B43]')
[2024-11-12T22:50:21.249+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 149 for task get_api (name 'df' is not defined; 890)
[2024-11-12T22:50:21.271+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-11-12T22:50:21.309+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-12T22:50:21.312+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
